src/lib/sw_engine/tvgSwRenderer.cpp

```diff
@@ -112,8 +123,71 @@ struct SwTask : Task
     end:
         shapeDelOutline(&shape, tid);
     }
+
+    void run(unsigned tid) override
+    {
+       prepareTask(tid);
```

Comment:
you can perform here immediately without prepareTask().

Reply:
changed